Makefile

```diff
@@ -170,6 +170,8 @@ check:
 	@! ineffassign . | grep -vF gossip/gossip.pb.go
 	@echo "errcheck"
 	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
+	@echo "returncheck"
+	@returncheck $(PKG)
```

Comment:
I think some of the tools don't take package-spec arguments. Yes, it's
inconsistent, but the best we can do for now.

On Mon, Jan 25, 2016 at 11:51 AM, Peter Mattis <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/cockroachdb/cockroach/pull/3944#discussion_r50711181>:
>
> > @@ -170,6 +170,8 @@ check:
> >  	@! ineffassign . | grep -vF gossip/gossip.pb.go
> >  	@echo "errcheck"
> >  	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
> > +	@echo "returncheck"
> > +	@returncheck $(PKG)
>
> Some of the checks use PKG while others check all of the source. Doesn't
> that seem strange and inconsistent? Probably not worth fixing, though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/3944/files#r50711181>.
>